src/profiles/registry.json

```diff
@@ -0,0 +1,28 @@
+{
+    "genericProfiles" : [
+        "button-controller", 
+        "thumbstick-controller", 
```

Comment:
As discussed in issue #70 , these names are ambiguous about the presence of a grip button. Here is what I have implemented in Chrome, which avoids the ambiguity:
controller
touchpad-controller
thumbstick-controller
touchpad-thumbstick-controller
grip-controller
grip-touchpad-controller
grip-thumbstick-controller
grip-touchpad-thumbstick-controller

This covers all the possible xr-standard Gamepads by specifying whether or not each optional element (grip, touchpad, thumbstick) are present. The trigger button will always be present, so it's not required in the name. JS code could even parse these generic strings to figure out the basic functionality of the controller without even needing to query a registry.

Reply:
I'm supportive of making the presence of a grip button part of the generic profiles. Initially I was worried about a combinatorial explosion of profile strings, but if we limit it to just the `xr-standard` inputs and list them in the same order each time (as Jacob shows) then it seems very manageable and useful to developers. I have no opinon on the ordering of elements other than to say I'd be fine with the above list as-is. (With the addition of `button-controller` as we originally had.)

Reply:
Is `button controller` supposed to mean a controller with just a trigger button (but no grip button)?

And yes, I agree we should limit the generic profiles to covering functionality from xr-standard inputs. If developers want to know about additional optional buttons (such as Oculus touch A/B buttons), then they need to look at the more specific profile strings.

Reply:
Yes, `button-controller` would be a device with only a trigger button (or only a primary button, even if it wasn't trigger-shaped.) It would basically be the fallback for an `xr-standard` device where none of the other generic profiles applied. The only device I'm aware of that it would apply to off the top of my head is the [HoloLens clicker](https://docs.microsoft.com/en-us/windows/mixed-reality/hardware-accessories#hololens-clicker), but regardless of how common it is it's a decen idea to have a clear plan for how to address any devices that fit that description.